File: OneSignalSDK/onesignal/src/main/java/com/onesignal/outcomes/OSOutcomeEventsV1Service.java
package com.onesignal.outcomes;

import com.onesignal.OneSignalAPIClient;
import com.onesignal.OneSignalApiResponseHandler;

import org.json.JSONObject;

class OSOutcomeEventsV1Service extends OSOutcomeEventsClient {

    OSOutcomeEventsV1Service(OneSignalAPIClient client) {
        super(client);
    }

    /***
     * API endpoint /api/v1/outcomes/measure
     */
    public void sendOutcomeEvent(JSONObject object, OneSignalApiResponseHandler responseHandler) {
        client.post("outcomes/measure", object, responseHandler);
    }
}

File: OneSignalSDK/onesignal/src/main/java/com/onesignal/outcomes/data/OSOutcomeEventsV1Service.kt
package com.onesignal.outcomes.data

import com.onesignal.OneSignalAPIClient
import com.onesignal.OneSignalApiResponseHandler
import org.json.JSONObject

internal class OSOutcomeEventsV1Service(client: OneSignalAPIClient) : OSOutcomeEventsClient(client) {
    /***
     * API endpoint /api/v1/outcomes/measure
     */
    override fun sendOutcomeEvent(jsonObject: JSONObject, responseHandler: OneSignalApiResponseHandler) {
        client.post("outcomes/measure", jsonObject, responseHandler)
    }
}